demos/twa-location-delegation/README.md

```diff
@@ -0,0 +1,4 @@
+# Trusted Web Activity / Location Delegation Demo
+
+This demo application shows how to make the TWA app uses Android location permission instead of
+browser permission.
```

Comment:
Can we add some pointers to the chunks that are relevant to location delegation? It looks to me the necessary steps for a developer who wants to enable this are:

1. Adding the location delegation dependency on build.gradle
2. Creating the LocationDelegationService
3. Adding a reference to LocationDelegationService on the AndroidManifest.xml

Reply:
Sounds good. Added.